### DISCUSSION TITLE
v20.10.1

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#4114] [Releases]
**Security fixes**
* placeholder, will be updated when cve is assigned and security advisory is published
* placeholder, will be updated when cve is assigned and security advisory is published

## What's Changed
* Fixed null parameter warning in Mage_Wishlist_Block_Abstract by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/4068
* Fixed null parameter warning in Mage_Shipping_Model_Resource_Carrier_Tablerate by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/4073
* Updated TinyMCE to 6.8.4 by @fballiano in https://github.com/OpenMage/magento-lts/pull/4084
* fix codestyle - empty line by @Flyingmana in https://github.com/OpenMage/magento-lts/pull/4112
* Fixed UI bug in Newsletter Templates backend page, filters not saved in session by @eneiasramos in https://github.com/OpenMage/magento-lts/pull/4077
* Added check for isModuleOutputEnabled('Mage_Review') in customer's backend page by @eneiasramos in https://github.com/OpenMage/magento-lts/pull/4075
* Fixed null parameter warning in Mage_Wishlist_Controller_Abstract::allcartAction() by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/4083
* Fixed null parameter warning in Mage_ImportExport_Model_Import_Entity_Product::_filterRowData() by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/4086
* Fixed invisible configuration values in backend by @fballiano in https://github.com/OpenMage/magento-lts/pull/4085
* Added hadling of allowed_extensions for file uploads in backend's configuration section by @eneiasramos in https://github.com/OpenMage/magento-lts/pull/4078
* Backend: Improved "copy button" style in legacy admin theme by @ma4nn in https://github.com/OpenMage/magento-lts/pull/4072
* Missing translations in Mage_Sales.csv by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/4110

**Full Changelog**: https://github.com/OpenMage/magento-lts/compare/v20.10.0...v20.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.10.1'>v20.10.1</a>.</em>

**Comment by alexh-swdev** [comment]
Just to clarify: Is this a security release or not? (Bc. of the empty "security fixes" block)

**Reply by Flyingmana** [reply]
yes it is

**Comment by tmewes** [comment]
Thanks for the release! Could you please adjust the version number in `app/Mage.php:217`?

**Reply by Flyingmana** [reply]
not without another release :(
will do another one in a few days with correct version noted in there

**Reply by tmewes** [reply]
Okay, thank you! :)

**Reply by Flyingmana** [reply]
got (hopefully) resolved with the v20.10.2 release now

**Reply by tmewes** [reply]
Yes, now the version is updated correctly. Thank you! :)

**Reply by empiricompany** [reply]
main branch still have patch 0 ?
https://github.com/OpenMage/magento-lts/blob/cabdc8bedb434c698c316ddbc6fe9c314c387fa7/app/Mage.php#L217
